go.mod

```diff
@@ -5,12 +5,12 @@ go 1.22.7
 require (
 	github.com/AlecAivazis/survey/v2 v2.3.7
 	github.com/Netflix/go-expect v0.0.0-20220104043353-73e0943537d2
-	github.com/canonical/lxd v0.0.0-20241022112222-538639622c38
+	github.com/canonical/lxd v0.0.0-20241106165613-4aab50ec18c3
 	github.com/canonical/microceph/microceph v0.0.0-20241021172252-2dd856d64c62
-	github.com/canonical/microcluster/v2 v2.0.4
-	github.com/canonical/microovn/microovn v0.0.0-20241018151920-4b82d2983810
-	github.com/charmbracelet/lipgloss v0.13.0
-	github.com/creack/pty v1.1.23
+	github.com/canonical/microcluster/v2 v2.0.5
+	github.com/canonical/microovn/microovn v0.0.0-20241101125123-0d5d663f6575
```

Comment:
we should update the makefile update-gomod to target microovn and microceph LTS branches